packages/react-components/react-spinbutton/src/stories/SpinButton/index.stories.tsx

```diff
@@ -20,7 +21,7 @@ const meta: Meta = {
   parameters: {
     docs: {
       description: {
-        component: [descriptionMd, bestPracticesMd].join('\n'),
+        component: [accessibilityMd, descriptionMd, bestPracticesMd].join('\n'),
```

Comment:
As much as I think a11y is important, I think the component description (and possibly the more concise best practices) should go before the a11y docs.
```suggestion
        component: [descriptionMd, bestPracticesMd, accessibilityMd].join('\n'),
```